docs/appnotes/README.md

```diff
@@ -0,0 +1,22 @@
+# Application Notes
+
+This folder contains applciation notes in markdown format.
+These documents provide ancillary information about the concepts in the TAMS API, and their application to real-world problems.
+No part of these documents are considered part of the API specification, although including specification-type language as examples, or "opt-in" extensions is allowed.
+
+There is a simple template `appnote-template.md` which describes the few restrictions there are on Application Notes.
+Select a unique number for your Applcation Note, and try to keep them at least somewhat close together.
+
+## Versioning
+
+It is anticipated that these notes will change over time, therefore we need a way to refer to specific versions.
+As these notes are stored in a git repository, we will make use of git commit IDs from the main branch to refer to a specific version.
+For those unfamiliar with git, if you want to check if the versions you are looking at is the same as one referred to by someone else, the command
+
+```shell
+git diff <commit> <commit> -- <filename>
+```
+
+will indicate if there are differences between the verions or not.
```

Comment:
On GitHub, if you push `y` while looking at e.g. https://github.com/bbc/tams/tree/andrewg-appnotes/docs/appnotes it'll fill in the full commit ID too - not sure if that's worth hinting at in the doc as well.